rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
Yes, that was the idea behind switching to a different macro. I wanted to keep the convention of using hyphens in `foreman-*` directory names in etc and var. But, perhaps `package_name` was a poorly chosen macro name. I'm good with `/etc/foreman_maintain` too if there's no opposition.